src/tools/Makefile.am

```diff
@@ -101,51 +117,76 @@ dnie_tool_LDADD = $(OPTIONAL_OPENSSL_LIBS)
 gids_tool_SOURCES = gids-tool.c util.c
 gids_tool_LDADD = $(OPTIONAL_OPENSSL_LIBS)
 
-npa_tool_SOURCES = npa-tool.c fread_to_eof.c util.c npa-tool-cmdline.c
+npa_tool_SOURCES = npa-tool.c fread_to_eof.c util.c
+nodist_npa_tool_SOURCES = npa-tool-cmdline.c
```

Comment:
should this be nodist? I would say that we should generate these files during dist and include them in the release tarball, similarly as we include the `configure`.